app/serializers/serializer_post_processing.rb

```diff
@@ -0,0 +1,18 @@
+# TODO: docs
+module SerializerPostProcessing
+  # TODO: docs
+  def serializable_object(options={})
+    super.tap {|hash|
+      modify_serializable_object(hash)
+    }
+  end
+
+  # TODO: docs
+  def modify_serializable_object(hash)
+    # TODO: having this method as an abstract overridable method is not very good,
```

Comment:
There is a problem that `serializable_object` method is a part of undocumented API of `ActiveModel::Serializer` and it is likely to change (in fact in AMS 0.10 it changes it's name to `serializable_hash`). That's why I think it is a good idea to keep the notion of this method and the behavior we implement on top of it separately and explicitly, so that we don't have to hunt for references to the abandoned method once we've moved to another version.

But maybe I can do something like the following here:
```
  def serializable_object(options={})
     modify_serializable_object(super)
   end

def modify_serializable_object(hash)
  hash
end
```

What do you think?